### DISCUSSION TITLE
How to fetch data from Mongodb database using Node.js?

### DISCUSSION THREAD

**Discussion by bycixman35** [nuxt/nuxt#22903] [Questions]
![image](https://github.com/nuxt/nuxt/assets/61555264/769717cb-ff96-4762-8330-6768ed8afdcb)

I have such data in my database. How can I fetch data with Freelancer Id is equal to 64ea26ba26c04b13209eeaa2?

 new AApoOrderService().list({
            situation: 2,
         "details.freelancer_id": req.user.name,
       
        
    }, { user_id: 0, updatedAt: 0,
        details: {
            $elemMatch: {
                freelancer_id: req.user.name
            }
        }
      
    }, { createdAt: -1 })

I wrote the above code and it only returned me first values

**Comment by ruveydayilmaz** [comment]
I don't know the details about your AApoOrderService, but if you were to use Mongoose, it would look like this:

```Javascript
const conditions = {
    situation: 2,
    'details': {
    $elemMatch: {
        freelancer_id: req.user.name, // I assume it's "64ea26ba26c04b13209eeaa2"
    },
    },
};

const projection = {
    user_id: 0,
    updatedAt: 0,
};

const options = {
    sort: { createdAt: -1 },
};

const results = await YourModel.find(conditions, projection, options);
```
